samples/dotnet/kernel-syntax-examples/Example22_a_OpenApiSkill_AzureKeyVault.cs

```diff
@@ -9,7 +9,7 @@
 using RepoUtils;
 
 // ReSharper disable once InconsistentNaming
-public static class Example22_OpenApiSkill
+public static class Example22_a_OpenApiSkill_AzureKeyVault
```

Comment:
the name seems off, we don't use "a" and "b"...

Reply:
This is to keep all the OpenApi Skills next to one another in the list. I dont think we should rename all the examples after them cause we might add more example variations. 
Could change it to Example_22_OpenApi.... but then it feels like a typo because Example_22_OpenApi.... shows up twice

Reply:
@dluc thoughts?

Reply:
We could just keep them all Example22 with the specific postfix 
Example22_OpenApiSkill_AzureKeyVault
Example22_OpenApiSkill_Jira
Example22_OpenApiSkill_GitHub